atlp-rwanda/e-commerce-lydia-32-bn
# Ch coveralls code coverage 187570853

What does this PR do?
Have code coverage uploaded to coveralls 

Description of Task to be completed?
Have installed all the dependencies necessities
Configuring circleci job for coverage uploading

How should this be manually tested?
After cloning  the repo, navigate to it and Run yarn coverage to see the coverage, write some tests, commit and push codes for circleci to upload them on the coveralls.io repo

 What are the relevant pivotal tracker stories

#187570853
